cibuildwheel/environment.py

```diff
@@ -84,6 +84,9 @@ def as_dictionary(
 
         return environment
 
+    def add(self, name: str, value: str) -> None:
+        self.assignments.append(EnvironmentAssignment(f'{name}="{value}"'))
```

Comment:
It would be worth a unit test with a double quote and maybe a trailing backslash to check we're quoting properly. But I _think_ this should be `f'{name}={shlex.quote(value)}'`.

Reply:
Ahh! I just found that docker passes environment variables without an equals by taking the value from the environment directly. Nice! I can use that instead of trying to quote things correctly.